docs/advanced-entity-analytics/entity-store.asciidoc

```diff
@@ -0,0 +1,52 @@
+[[entity-store]]
+= Entity store
+
+preview::[]
+
+.Requirements
+[sidebar]
+--
+To use the entity store, you must have the appropriate privileges. For more information, refer to <<ers-requirements, Entity risk scoring requirements>>.
+--
+
+The entity store allows you to query, reconcile, maintain, and persist entity metadata from various sources such as:
+
+* Ingested log sources
+* Integrated identity providers (such as Active Directory, EntraID, and Okta)
+* Internal and external alerts
+* External asset repositories
+* Asset criticality submissions
+
+The entity store can store any entity type observed by {elastic-sec}. This allows you to store, view, and query select entities directly from an index without needing to perform real-time searches of observable data. The entity store extracts entities from all indices in the {elastic-sec} <<default-data-view-security, default data view>>.
+
+When the entity store is enabled, the following resources are generated:
+
+* {es} resources, such as transforms, ingest pipelines, and enrich policies.
+* Data and fields for each entity.
+* The `.entities.v1.latest.security_user_<space-id>` and `.entities.v1.latest.security_host_<space-id>` indices, which contain field mappings for hosts and users respectively. You can query these indices to see a list of fields that are mapped in the entity store.
+
+[discrete]
+[[enable-entity-store]]
+== Enable entity store
+
+To enable the entity store:
+
+. Find **Entity Store** in the navigation menu or by using the {kibana-ref}/introduction.html#kibana-navigation-search[global search field].
+. On the **Entity Store** page, turn the toggle on. 
+
+Once you enable the entity store, the Entity Analytics dashboard displays the <<entity-entities, **Entities**>> section.
+
+[discrete]
+[[clear-entity-store]]
+== Clear entity store data
+
+Once the entity store is enabled, you may want to start fresh and clear the extracted entity data. For example, if you have refined your data by normalizing the `user.name` or `host.name` fields, clearing the entity store data allows you to repopulate the entity store with the updated, normalized values. This action removes all previously extracted entity information, enabling new data extraction and analysis.
```

Comment:
```suggestion
Once the entity store is enabled, you may want to clear existing data and start fresh. For example, if you normalized the `user.name` or `host.name` fields, clearing the entity store data would allow you to repopulate the entity store with the updated, normalized values. This action removes all previously extracted entity information, enabling new data extraction and analysis.
```